### PR TITLE
Dropping support for node 8

### PR DESCRIPTION
**Pull request by scalvert**
This PR also removes the redundant .travis.yml.

### CODE REVIEW

**Comment by scalvert** [comment]
We should also wait for #96 to merge before releasing, so we can have nicer changelogs :)

**Comment by scalvert** [comment]
Going to wait for #86 to be merged and released before merging and releasing this in a major version bump.

**Comment by scalvert** [comment]
@rwjblue I think since #86 is merged we can think about merging this and releasing. Thoughts?

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "**/engine.io": "~3.3.0"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": ">= 10.*"
```
Comment (rwjblue) [inline package.json]:
```suggestion
    "node": "10.* || >= 12.*"
```

File: package.json [hunk 2]
```diff
@@ -74,7 +74,7 @@
     "**/engine.io": "~3.3.0"
```
Comment (rwjblue) [inline package.json]: I think we can remove this

File: .github/workflows/ci-build.yml [hunk 3]
```diff
@@ -61,7 +61,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
         with:
-          node-version: 12.x
```
Comment (rwjblue) [inline .github/workflows/ci-build.yml]: Let's keep this using 12.x

File: package.json (hunk 2 above)
Comment (scalvert) [reply package.json]: Done

File: .github/workflows/ci-build.yml (hunk 3 above)
Comment (scalvert) [reply .github/workflows/ci-build.yml]: Updated

**Pull request merged**